mlflow/store/artifact/http_artifact_repo.py

```diff
@@ -113,11 +113,19 @@ def delete_artifacts(self, artifact_path=None):
         resp = http_request(self._host_creds, endpoint, "DELETE", stream=True)
         augmented_raise_for_status(resp)
 
+    def _mpu_path(self, base_endpoint, artifact_path):
+        uri, path = self.artifact_uri.split("/mlflow-artifacts/artifacts", maxsplit=1)
```

Comment:
Is it guaranteed that we always have the `/artifacts` suffix?

Reply:
`self.artifact_uri` dose. It should looks like `.../mlflow-artifacts/artifacts/<experiment_id>/<run_id>`.

Reply:
Ah I see, thanks for the clarification!